Basics of JS/2.3/Closures_And_Scopes.js

```diff
@@ -0,0 +1,9 @@
+function createFunctions(n){
+    var callbacks = [];
+
+    for(let i=0;i<n;i++){
+        callbacks.push(function(){
+            return i;
+        });
+    }
```

Comment:
return callback array

Reply:
sorry Garv must have missed by mistake. Have corrected it in the [e8bffe1](https://github.com/rajkumar-telu/HomeTasks-Epam/pull/2/commits/e8bffe13f545117b31ecf4e796afa27566ee6899) commit.